src/QueryLanguage/Processor/Doctrine/AbstractProcessor.php

```diff
@@ -197,6 +199,24 @@ private function resolveOptions(array $options): array
 
                 return $value;
             })
+            ->setNormalizer('default_order', static function (Options $options, $value): ?OrderExpression {
+                if (empty($value)) {
+                    return null;
+                }
+
+                if (false === \strpos($value, '$')) {
+                    $value = '$order('.$value.')';
+                }
+
+                $grammar = new Grammar();
```

Comment:
Instead of `new Grammar()` I would use the `Grammar::getInstance()`. Probably it could be interesting overriding that grammar. Using `$this->grammar` (and giving possibility to override that with a setter `setGrammar(?Grammar $grammar)`) could make this possible (like in the transformer above)

Reply:
Done